.github/workflows/ci.yml

```
@@ -57,31 +57,25 @@ jobs:
         with:
           toolchain: nightly
-          components: miri
 
       - name: Update Rust nightly
         run: rustup update nightly
 
-      - name: Install Miri
-        run: rustup component add miri --toolchain nightly
-
-      - run: make miri
+      - name: Run Unit Test
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)